src/pages/Add-items/Add-items.js

```diff
@@ -76,82 +79,95 @@ export const AddItems = () => {
-                checked={product.howSoon === '7'}
-                onChange={handleChangeProduct}
-              />
-              Soon
-            </label>
-          </div>
-          <div>
-            <label htmlFor="kindOfSoon">
-              <input
-                id="kindOfSoon"
-                type="radio"
-                name="howSoon"
-                required
-                value="14"
-                checked={product.howSoon === '14'}
-                onChange={handleChangeProduct}
-              />
-              Kind of soon
-            </label>
-          </div>
-          <div>
-            <label htmlFor="notSoon">
+    <main>
+      <form className="filter-form-add-items" onSubmit={handleSubmit}>
+        <div className="list-header-add-items">
+          <label htmlFor="name">
+            Add new products to your list:
+            <div className="filter-item inputField">
               <input
-                id="notSoon"
-                type="radio"
-                name="howSoon"
                 required
-                value="30"
-                checked={product.howSoon === '30'}
+                id="name"
+                className="filter-input"
+                type="text"
+                name="name"
+                value={product.name}
                 onChange={handleChangeProduct}
+                ref={inputRef}
+                placeholder="New product"
               />
-              Not soon
-            </label>
-          </div>
-        </fieldset>
+            </div>
+          </label>
+
+          <p className="how-soon">How soon will you buy this again?</p>
+          <fieldset>
+            <div>
+              <label htmlFor="soon">
+                <input
+                  id="soon"
+                  type="radio"
+                  name="howSoon"
+                  required
+                  value="7"
+                  checked={product.howSoon === '7'}
+                  onChange={handleChangeProduct}
+                />
+                Weekly
+              </label>
+            </div>
+            <div>
+              <label htmlFor="kindOfSoon">
+                <input
+                  id="kindOfSoon"
+                  type="radio"
+                  name="howSoon"
+                  required
+                  value="14"
+                  checked={product.howSoon === '14'}
+                  onChange={handleChangeProduct}
+                />
+                Biweekly
+              </label>
+            </div>
+            <div>
+              <label htmlFor="notSoon">
+                <input
+                  id="notSoon"
+                  type="radio"
+                  name="howSoon"
+                  required
+                  value="30"
+                  checked={product.howSoon === '30'}
+                  onChange={handleChangeProduct}
+                />
+                Monthly
+              </label>
+            </div>
+          </fieldset>
```

Comment:
❓  Pregunta:

¿Cuál fue la razón que los motivo a cambiar de `soon`, `kind of soon`, y `not soon` a `weekly`, `biweekly` y `monthly`?

Nota.- Siempre es importante, respetar los **criterios de aceptación** 👀

Reply:
De acuerdo a lo que conversamos, creimos importante serle mas claro al usuario sobre sus opciones de marcado. No sabiamos que debia considerarse mantener ese AC, resuelto  ***c6231c109871a2ca93e66bfa807f4bb0e7f23644*** listo!

Reply:
Está bien, introducir estos cambios o **mejores** que ayuden al usuario 💯 !!!

Jajaja 🤣 😄  ... por eso, que tampoco yo sugerí que lo **"retornen a lo anterior"**, sino que era un consulta, entender los motivos. Recuerden todos que si yo soy un usuario habitual de la app y de pronto cambia un descripción, botón o alguna otra cosa, me puede generar confusión. 👁️ 

Este tipo de cosas, se sugieren preguntar al product owner. 😃 Estrategias de comunicar el cambio a todos los usuarios.